.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+versions:
+  - "2.7"
+  - "3.5"
+cache: pip
+services:
+  - docker
+
+install:
+  - pip install -U pip
+  - pip --version
+  - pip install ansible ansible-lint yamllint flake8 molecule docker
+  - ansible --version
+  - ansible-lint --version
+  - yamllint --version
+  - flake8 --version
+  - molecule --version
+
+script:
+  - molecule test -s connection_only
```

Comment:
If we're only using the `connection_only` scenario, do we also need the `docker` one down there?

Reply:
Having the docker create/destroy books split out is handy for making it easier to write additional docker scenarios, but it's easy enough to split out later that I'll get rid of it until/unless we actually develop another scenario depending on the docker bits.